packages/babel/cli.js

```diff
@@ -1,10 +1,13 @@
 #!/usr/bin/env node
+const isGlobal = require("is-global");
+
+const globalMessage = isGlobal() ? " -g" : "";
 
 console.error("You have mistakenly installed the `babel` package, which is a no-op in Babel 6.\n" +
   "Babel's CLI commands have been moved from the `babel` package to the `babel-cli` package.\n" +
   "\n" +
-  "    npm uninstall babel\n" +
-  "    npm install babel-cli\n" +
+  "    npm uninstall" + globalMessage + " babel-cli\n" +
```

Comment:
I think there was a miscommunication, having this be `babel` and not `babel-cli` is on purpose, because `babel` is the package that contains the script that prints this warning, and we want to tell people to remove it.